Source/sound.cpp

```diff
@@ -306,4 +306,16 @@ int sound_get_or_set_sound_volume(int volume)
 	return sgOptions.Audio.nSoundVolume;
 }
 
+void music_mute()
+{
+	if (music)

```

Comment:
@AJenbo weren't we using explicit checks?

Reply:
yeah, not sure what type this is, but clang-tidy should get it if it's not a bool